src/Umbraco.Core/Models/PropertyTagsExtensions.cs

```diff
@@ -23,19 +23,15 @@ public static class PropertyTagsExtensions
 
         IDataEditor? editor = propertyEditors[property.PropertyType?.PropertyEditorAlias];
         TagsPropertyEditorAttribute? tagAttribute = editor?.GetTagAttribute();
-        if (tagAttribute == null)
-        {
-            return null;
-        }
 
         var configurationObject = property.PropertyType is null
             ? null
             : dataTypeService.GetDataType(property.PropertyType.DataTypeId)?.Configuration;
-        TagConfiguration? configuration = ConfigurationEditor.ConfigurationAs<TagConfiguration>(configurationObject);
+        TagConfiguration? configuration = configurationObject as TagConfiguration;
 
         if (configuration is not null && configuration.Delimiter == default)
         {
-            configuration.Delimiter = tagAttribute.Delimiter;
+            configuration.Delimiter = tagAttribute?.Delimiter ?? ',';
```

Comment:
@rasmusjp maybe use the comma from the constant class here?

Reply:
https://github.com/umbraco/Umbraco-CMS/blob/b69afe81f3f6fcd37480b3b0295a62af44ede245/src/Umbraco.Core/Constants-CharArrays.cs#L53

but since it is an array we would need to select first char.